docs/reference/reana-yaml/index.md

```diff
@@ -112,6 +113,16 @@ The **workflow.specification.steps** property describes each individual computat
 | environment  | *string*  | *mandatory*  | Specifies runtime environment container image where the given workflow step commands will be run.  |
 | commands  | *list*  | *mandatory*  | Lists all commands to be run in the runtime environment container image when the given workflow step is executed. Note that each command is executed as a separate containerised job. |
 
+The **workflow.options** property describes operational options that can be used for the different workflow engines. The available options are:
+
+| Property | Type     | Mandatory?  | Workflow engine | Description                            |
+| -------- | ----     | ----------  | ----------------|                                        |
+| CACHE    | *string* | *optional*  | Serial          | Whether the workflow engine should cache the results of each step for faster execution of subsequent workflow runs. Disabled by default. Can be `on` or `off`. |
+| FROM     | *string* | *optional*  | Serial          | Allows partial execution of a workflow from the `FROM` step, including it. It can be combined with `TARGET`. The value must be the name of the step. |
+| TARGET   | *string* | *optional*  | Serial, CWL     | Allows partial execution of a workflow until the `TARGET` step, including it. It can be combined with `FROM`. The value must be the name of the step. |
+| toplevel | *string* | *optional*  | Yadage          | Yadage `toplevel` argument. It represents the working directory or remote repository where the workflow should be pulled from. [More info](https://yadage.readthedocs.io/en/latest/definingworkflows.html?highlight=toplevel#using-json-references). |
```

Comment:
Mention that we support GitHub repos ([only those for the moment!](https://github.com/reanahub/reana-workflow-engine-yadage/blob/master/reana_workflow_engine_yadage/cli.py#L48-L51)) and add an example of the format `github:<username/repo[@branch]>[:subpath]`